addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -146,8 +149,8 @@ sitemap demo label="myKodi"
         Text      item=myKodi_showtitle
         Text      item=myKodi_album
         Text      item=myKodi_artist
-        Selection item=myKodi_pvropentv mappings=[Add your PVR TV channels here ...]
-        Selection item=myKodi_pvropenchannel mappings=[Add your PVR radio channels here ...]
+        Selection item=myKodi_pvropentv
+        Selection item=myKodi_pvropenchannel
```

Comment:
Since this depends on [this feature](https://github.com/eclipse/smarthome/issues/4942) being implemented in ESH, is the intention to delay the merge of this until that feature is part of a future ESH stable?

Also, WDYT about a HABpanel Selection widget example, too?

Reply:
Yes and No. The example was bad before. But since it depends on the users configuration of Kodi I can't think of a better one.

> WDYT about a HABpanel Selection widget example

Hm, ... I wouldn't support that. IMHO the documentation should be independent and not related to any UI. Shouldn't it?

Reply:
Ideally, yes, the documentation should be independent of the UI. OTOH, the downside is that it forces the user to figure it out on their own, or to hunt on the forum for someone who's already done it.

My motive for asking was purely selfish, as I was struggling with this exact question for the squeezebox favorites. ;-)